cob_image_flip/ros/src/image_flip.cpp

```diff
@@ -311,14 +378,14 @@ void ImageFlip::imageCallback(const sensor_msgs::ImageConstPtr& color_image_msg)
 	cv::Mat rot33 = cv::Mat::eye(3,3,CV_64FC1);
 	for (int r=0; r<2; ++r)
 		for (int c=0; c<3; ++c)
-			rot33.at<double>(r,c) = rot_mat.at<double>(r,c);
```

Comment:
Just leave it the old way in this case, because the rotation matrix is always the same format CV_64FC1.